webrtc.html

```diff
@@ -7339,6 +7340,12 @@ <h2>Dictionary RTCRtpContributingSource Members</h2>
               127 is converted to 0, and all other values are converted using
               the equation: <code>10^(-rfc_level/20)</code>.</p>
             </dd>
+            <dfn data-idl><code>rtpTimestamp</code></dfn> of type <span class=
+            "idlMemberType">unsigned long</span>
+            <dd>
+              <p>The RTP timestamp of the latest played out media that arrived
+              in an RTP packet originating from this source.</p>
```

Comment:
Can we refer this to the variable above, as "RTP timestamp of the media played out at "timestamp", to make it clear that the two timestamps are required to be from the same packet?

Reply:
I agree with @alvestrand that this needs to correlate with the same packet, not the "latest".

See also https://github.com/w3c/webrtc-pc/issues/2172#issuecomment-486740788 wrt this API—at least in Firefox—returns data no sooner than at time of playout.

Reply:
Yes, that makes sense. I think this change should be coordinated with #2172 .

Reply:
Done.